modules/github-bots/sdk/bot.go

```diff
@@ -128,13 +128,7 @@ func Serve(b Bot) {
 					return err
 				}
 
-				err = h(ctx, wre.Body, wr)
-				if err != nil {
-					logger.Errorf("failed to handle workload run event: %v", err)
```

Comment:
Does the top level bot report the error if we don't log it here?

Reply:
Looks like we can report in each bots invocation of `Serve`, but we don't right now so this changes the error logging for existing bots